### PR TITLE
docs: follow up on adding trace dir, unify launch options

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: docs/src/api/class-tracing.md [hunk 1]
```diff
@@ -1,7 +1,50 @@
...
+const context = await browser.newContext();
+await context.tracing.start({ name: 'trace', screenshots: true, snapshots: true });
+const page = await context.newPage();
+await page.goto('https://playwright.dev');
+await context.tracing.stop();
+await context.tracing.export('trace.zip');
+```
+
+```java
+Browser browser = chromium.launch(new BrowserType.LaunchOptions().setTraceDir("trace"
```
Comment (dgozman) [inline docs/src/api/class-tracing.md:19]: This line is truncated.

File: types/types.d.ts [hunk 2]
```diff
@@ -10882,7 +10906,7 @@ export interface LaunchOptions {
   channel?: "chrome"|"chrome-beta"|"chrome-dev"|"chrome-canary"|"msedge"|"msedge-beta"|"msedge-dev"|"msedge-canary"|"firefox-stable";
 
   /**
-   * Enable Chromium sandboxing. Defaults to `false`.
+   * Enable Chromium sandboxing. Defaults to `true`.
```
Comment (dgozman) [inline types/types.d.ts:10909]: Why did this change?

**Review by dgozman — APPROVED** [review]

File: types/types.d.ts (hunk 2 above)
Comment (pavelfeldman) [reply types/types.d.ts:10909]: There was a bug I think...

**Comment by kblok** [comment]
@pavelfeldman, @dgozman  My two cents here. Let's remember that a change in the arguments order could be considered a breaking change.

**Pull request merged**